src/picongpu/include/particles/creation/creation.kernel

```diff
@@ -0,0 +1,305 @@
+/**
+ * Copyright 2015-2016 Heiko Burau
```

Comment:
This file contains a lot of code from the file `ionization.hpp` therefore please keep the original authors.